torch/onnx/symbolic_opset10.py

```diff
@@ -345,7 +346,7 @@ def _slice(
         if (
             len(starts) == 1
             and starts[0] == 0
-            and ends[0] == 9223372036854775807
+            and ends[0] == _INT64_MAX
```

Comment:
Is it only appearing in this file? Should `_INT64_MAX` reside in `_constants.py`?

Reply:
Sure! Moved to _constants